frontend/src/components/pages/consumers/Modals.tsx

```diff
@@ -605,7 +589,7 @@ class ColAfter extends Component<{
                                     There is no offset for this partition at or after the given timestamp (<code>{new Date(this.props.selectedTime ?? 0).toLocaleString()}</code>). As a fallback, the last offset in that partition will be used.
                                 </div>
                             }
-                            icon={<WarningOutlined/>}
+                            icon={<MdOutlineWarningAmber size={14} />}
```

Comment:
Any reason why we have different icon sizes in various locations? I think 16 is the default?